AsyncDisplayKit.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |spec|
       'Base/*.{h,m}'
   ]
 
-  spec.frameworks = 'AssetsLibrary'
+  spec.frameworks = 'AssetsLibrary','MapKit'
   spec.weak_frameworks = 'Photos'
```

Comment:
@appleguy happy to change this to weak if that's an option, will wait to hear back from @Adlai-Holler